### PR TITLE
[BD-21] Fix github link in setting docs

### PR DESCRIPTION
**Pull request by regisb**
**Description:** The "filename" value returned by a static search is relative to the
AnnotationConfig.source_path attribute. As a consequence, when we
used the sphinx extension to document settings with `:folder_path:
lms/envs/common.py` this resulted in incorrect links, such as:
https://github.com/edx/edx-platform/blob/97e9fee92ed2c553e6a669f8ffba9151b122129d/common.py#L57

Compare to the correct link:
https://github.com/edx/edx-platform/blob/97e9fee92ed2c553e6a669f8ffba9151b122129d/lms/envs/common.py#L57

**JIRA:** https://openedx.atlassian.net/wiki/spaces/COMM/pages/1596358943/BD-21+Toggles+Settings+Documentation

**Reviewers:**
- [ ] @robrap 

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green
- [x] Version bumped
- [x] Changelog record added
- ~~[ ] Documentation updated (not only docstrings)~~
- [x] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPi after tag-triggered build is
      finished.
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @regisb! I've created [BLENDED-660](https://openedx.atlassian.net/browse/BLENDED-660) to keep track of it in Jira.   More details are on the [BD-21](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1596358943/BD-21+Toggles+Settings+Documentation) project page.


When this pull request is ready, tag your edX technical lead.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -11,6 +11,12 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
```
Comment (robrap) [inline CHANGELOG.rst:13]: Nit: 0.10.1 would be for bug fix.

**Review by robrap — APPROVED** [review]
Are there tests? Do we want to enhance at some point? For now, I’m just relying on readthedocs updates.

**Comment by regisb** [comment]
There are no unit test (yet) for the sphinx extensions... I'll add a follow-up task to remind me to add some.

**Pull request merged**